tests/dhcp_relay/test_dhcp_relay.py

```diff
@@ -112,12 +118,31 @@ def validate_dut_routes_exist(duthosts, rand_one_dut_hostname, dut_dhcp_relay_da
         assert len(rtInfo["nexthops"]) > 0, "Failed to find route to DHCP server '{0}'".format(dhcp_server)
 
 
-def test_dhcp_relay_default(duthosts, rand_one_dut_hostname, ptfhost, dut_dhcp_relay_data, validate_dut_routes_exist):
+def set_dual_tor(duthost):
+    duthost.shell('redis-cli -n 4 HSET "DEVICE_METADATA|localhost" "subtype" "DualToR"')
+    duthost.shell('systemctl restart dhcp_relay')
+    duthost.shell('systemctl reset-failed dhcp_relay')
+    time.sleep(30)
+
+
+def reset_dual_tor(duthost):
```

Comment:
Suggest renaming to `clear_dual_tor_mode`

Reply:
updated, thanks